src/backend/cli/main.py

```diff
@@ -42,7 +42,7 @@ class ToolName(StrEnum):
 """
 DATABASE_URL_DEFAULT = "postgresql+psycopg2://postgres:postgres@db:5432"
 PYTHON_INTERPRETER_URL_DEFAULT = "http://localhost:8080"
-NEXT_PUBLIC_API_HOSTNAME_DEFAULT = "http://localhost:8000"
+NEXT_PUBLIC_API_HOSTNAME_DEFAULT = "http://backend:8000"
```

Comment:
We define this variable in multiple places, such as .env_template, README, and some FE files. Can you please update all the occurrences?

Reply:
I _think_ I've got it now in all the places where we need it. Outside of the containers, `localhost:8000` is still the right way to reach the backend (e.g. when using the Postman collections). We just want to make sure that the frontend container is using `backend:8000` when we're running locally with `docker compose`.